contributing/dev_env_setup.sh

```diff
@@ -0,0 +1,176 @@
+
+
+if [ $action = "--full" ]
+then
+######################################################
+# PACKAGE DEPENDENCIES INSTALLATION + HOME PRE-SETUP #
+######################################################
+
+    cd into a directory that is easily accessible by other users
+    sudo /usr/sbin/usermod -aG sudo $username
+    sudo apt update
+    # apt upgrade
+    sudo apt install -y git openssh-server python3 python3-pip python python-pip openjdk-11-jdk net-tools ldap-utils maven apache2
+    sudo service ssh start
+
+    sudo chown $username -R /home/gpoblon/
+
+    mkdir -p $rudder_dir
+    cd $rudder_dir
+
+###################################
+# REQUIRED SOFTWARES INSTALLATION #
+###################################
+
+    # install vagrant latest
+    vagrantversion=$(wget -qO- https://raw.githubusercontent.com/hashicorp/vagrant/stable-website/version.txt) 
+    wget -q https://releases.hashicorp.com/vagrant/${vagrantversion}/vagrant_${vagrantversion}_x86_64.deb
+    sudo dpkg -i vagrant_${vagrantversion}_x86_64.deb
+    sudo apt -f install
+    rm -rf vagrant_${vagrantversion}_x86_64.deb
+
+    # install virtualbox + extension pack
+    sudo add-apt-repository "deb http://download.virtualbox.org/virtualbox/debian bionic contrib"
+    wget -qO- https://www.virtualbox.org/download/oracle_vbox_2016.asc | sudo apt-key add -
+    sudo apt update
+    vboxlatest=$(wget -qO- https://download.virtualbox.org/virtualbox/LATEST.TXT)
+    vboxversion=$(echo $vboxlatest | cut -c1-3)
+    sudo apt install -y virtualbox-${vboxversion}
+    wget -q https://download.virtualbox.org/virtualbox/${vboxlatest}/Oracle_VM_VirtualBox_Extension_Pack-${vboxlatest}.vbox-extpack
+    echo y | sudo vboxmanage extpack install --replace Oracle_VM_VirtualBox_Extension_Pack-${vboxlatest}.vbox-extpack
+    rm -rf Oracle_VM_VirtualBox_Extension_Pack-${vboxlatest}.vbox-extpack
+
+    # install intelliJ
+    wget -q https://download.jetbrains.com/idea/ideaIC-2019.3.3.tar.gz
+    sudo tar -xzf ideaIC-2019.3.3.tar.gz -C /opt
+    rm -rf ideaIC-2019.3.3.tar.gz
+    # /opt/idea-IC-193.6494.35/bin/idea.sh # to open it
+
+    #install ldap apache directory studio
+    wget -q http://apache.mirrors.ovh.net/ftp.apache.org/dist/directory/studio/2.0.0.v20180908-M14/ApacheDirectoryStudio-2.0.0.v20180908-M14-linux.gtk.x86_64.tar.gz
+    sudo tar -xzf ApacheDirectoryStudio-2.0.0.v20180908-M14-linux.gtk.x86_64.tar.gz -C /opt
+    rm -rf ApacheDirectoryStudio-2.0.0.v20180908-M14-linux.gtk.x86_64.tar.gz
+    # /opt/ApacheDirectoryStudio/ApacheDirectoryStudio # to open it
+
+    # install Elm
+    wget -qO elm.gz https://github.com/elm/compiler/releases/download/0.19.1/binary-for-linux-64-bit.gz
+    gzip -d elm.gz
+    chmod +x elm
+    sudo mv elm /usr/local/bin/
+    rm -rf elm.gz
+
+###########################
+# RUDDER ENV INSTALLATION #
+###########################
+
+    # install rudder-tests environment (required for later use)
+    git clone https://github.com/gpoblon/rudder
```

Comment:
should probably use `rudder-dev clone`

Reply:
Though not every user has / uses rudder-dev ?

Reply:
would be easier with rudder-dev but would be easier for external people (or else we should add rudder-dev setup at the beginning of the contribute documentation

Reply:
that's what I thought, rudder-dev users can still take advantage of it, it's just a clone